src/porepy/numerics/linalg/matrix_operations.py

```diff
@@ -694,15 +699,15 @@ def inv_python(indptr, ind, data, sz):
 
     # Remove blocks of size 0
     s = s[s > 0]
-    # Variable to check if we have tried and failed with numba
-    if method == "numba" or method is None:
+    # Select python vectorized function
+    if method == "python" or method is None:
```

Comment:
I agree with making this the default, but to do so properly, we should also change the default value of the inverter in mpfa, mpsa (and possibly Biot).

Reply:
I have updated the default on those files.